1.0/Dockerfile

```diff
@@ -38,5 +28,9 @@ RUN /tmp/build.sh
 
 ENV LD_PRELOAD /opt/td-agent/embedded/lib/libjemalloc.so
 
+RUN /opt/fusion/bin/fusion start
```

Comment:
For convenience in testing. If this is used as a layered file in prod you wouldn't do this.